rrmngmnt/service.py

```diff
@@ -199,7 +199,14 @@ def _execute(self, action):
         ]
         executor = self.host.executor()
         rc, _, _ = executor.run_cmd(cmd, io_timeout=self.timeout)
-        return rc == 0
+
+        rc_result = rc == 0
+        if not rc_result:
+            cmd = ['journalctl', '-u', self.name]
+            _, out, _ = executor.run_cmd(cmd, io_timeout=self.timeout)
+            self.logger.info(out)
```

Comment:
I would consider warning or error log level if it's happening when rc 0. But as @StLuke  suggested I also would not add it here and rather check it in your side when you need it.  Otherwise you will need to cover all tests in tests/test_service.py an for all non zero output like  'systemctl status s-stopped.service': (1, '', ''),   you will need to put something like 'journalctl -u s-stopped.service': (0, 'Std out for this err', 'Errmsg'),

Reply:
regarding to tests, I added [test_docker.py](https://github.com/rhevm-qe-automation/python-rrmngmnt/blob/master/tests/test_docker.py) where you can run tests against containers.

So far I have ubuntu system there, but feel free to extend it to improve coverage.